packages/ui-components/src/Common/Badge/index.tsx

```diff
@@ -3,10 +3,11 @@ import type { FC, HTMLAttributes, PropsWithChildren } from 'react';
 
 import styles from './index.module.css';
 
-type BadgeKind = 'default' | 'warning' | 'error';
+type BadgeKind = 'default' | 'warning' | 'error' | 'info' | 'neutral';
```

Comment:
nit: We re-use this type in AlertBox, perhaps they should share it?

Reply:
Although similar, I believe they should be independent.